src/main/java/com/training/web/pages/flipkart/ResultPage.java

```diff
@@ -1,44 +1,38 @@
 package com.training.web.pages.flipkart;
 
+import com.codeborne.selenide.CollectionCondition;
+import com.codeborne.selenide.ElementsCollection;
+import com.codeborne.selenide.SelenideElement;
 import com.training.basepages.FlipkartBasePage;
-import org.openqa.selenium.By;
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.WebElement;
-import org.openqa.selenium.support.ui.ExpectedConditions;
-import org.openqa.selenium.support.ui.WebDriverWait;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import java.util.*;
+import static com.codeborne.selenide.Selenide.*;
+import static com.codeborne.selenide.Condition.*;
+import static com.codeborne.selenide.WebDriverRunner.getWebDriver;
 
 public class ResultPage extends FlipkartBasePage {
     private final Logger logger = LoggerFactory.getLogger(this.getClass());
 
-    private By lowToHigh = By.xpath("//*[text()='Price -- Low to High']");
-    private By productResults = By.xpath("//img[contains(@class,'_2r_T1I')]");
-    private By shoesPrice = By.className("_30jeq3");
-    private By nextPageButton = By.xpath("//*[text()='Next']");
-    private By products = By.xpath("//div[@class='_312yBx SFzpgZ']");
-
-    public ResultPage (WebDriver driver) {
-        super(driver);
-    }
+    private SelenideElement lowToHigh = $x("//*[text()='Price -- Low to High']");
+    private ElementsCollection productResults = $$x("//img[contains(@class,'_2r_T1I')]");
+    private ElementsCollection shoesPrice = $$x("//*[@class='_30jeq3']");
+    private static SelenideElement nextPageButton = $x("//*[text()='Next']");
+    private SelenideElement products = $x("//div[@class='_312yBx SFzpgZ']");
```

Comment:
Lets define this in a string rather than SelenideElement, and then use it in your methods